android/app/src/main/java/co/ello/ElloApp/MainActivity.java

```diff
@@ -391,7 +402,35 @@ public void onActionClicked(Snackbar snackbar) {
         registerReceiver(pushReceivedReceiver, new IntentFilter(ElloPreferences.PUSH_RECEIVED));
     }
 
+    private void setupRatePrompt(){
+        AppRate.with(this)
+            .setInstallDays(7)
+            .setLaunchTimes(3)
+            .setRemindInterval(7)
+            .setShowLaterButton(true)
+            .setDebug(true)
```

Comment:
Not one we would release to users but there is no other way to test it internally if we set it to false (we'd have to go through all the requirements before it was testable). We'll need to have it QA'd and then release a new build once it is approved.